### DISCUSSION TITLE
Storage 403 in Production

### DISCUSSION THREAD

**Discussion by trevormanhuwa** [laravel/framework#46054] [Q&A]
- Laravel Version: 9.44.0
- PHP Version: 8.1.13
- Database Driver & Version: mysql

### Description:

i have been having trouble with accessing my files in storage getting in production, online the files show and work well,

I have done everythinng, from tweaking the storage file,
php artisan storage link --relative
i have even updated htaccess files,

if you visit this link which is an asset link you get this annoying error.
http://lea-test-co-bw.stackstaging.com/storage/posters/1757345883467975.jpg

**Comment by denizgolbas** [comment]
You need to give permission.
https://stackoverflow.com/questions/30639174/how-to-set-up-file-permissions-for-laravel
